Gemfile

```diff
@@ -45,6 +45,7 @@ gem 'warden-basic_auth', '~> 0.2.1'
 gem 'rails_autolink', '~> 1.1.6'
 gem "will_paginate", '~> 3.0'
 gem "acts_as_list", "~> 0.3.0"
+gem 'acts_as_countable', git: "https://github.com/finnlabs/acts_as_countable.git"
```

Comment:
Should we specify any kind of version here?

I remember @ulferts saying that we should be more specific here...

I'd probably lock us to 0.1.x, i.e. only patch level releases. Even if we fully control the source gem.